specs/mgt-login.md

```diff
@@ -0,0 +1,124 @@
+# mgt-login
+
+The Login component provides a button and flyout to facilitate Microsoft Identity Platform authentication. The login component provides two states:
+- A sign in button when there are no users signed in
+- When a user or users are signed in, the control displays the display name and profile photo or avatar of the currently active account. When clicked, a flyout is rendered with additional profile data of all signed in accounts.
+
+Two versions of the Login component will exist (single account and multi-account). The version of the component rendered will be dependent on the authentication provider used by the developer. If the developer uses the component with a provider that supports multiple accounts (ex: MSAL 2.0), the flyout will include an option to add another account and display all currently signed in accounts.
```

Comment:
I think `versions` indicates there will be two different login components. I think you the last sentence here summarizes the multi user support nicely.

```suggestion
If the developer uses the component with a provider that supports multiple accounts (ex: MSAL 2.0), the flyout will include an option to add another account and display all currently signed in accounts.
```